setup.cfg

```diff
@@ -9,20 +9,17 @@ upload = upload upload_docs --upload-dir build/sphinx/html
 [bdist_rpm]
 doc-files = doc
 
-[nosetests]
-verbosity = 2
-detailed-errors = 1
-with-coverage = 1
-cover-package = joblib
-#pdb = 1
-#pdb-failures = 1
-with-doctest=1
-doctest-extension=rst
-doctest-fixtures=_fixture
-
 [wheel]
 universal=1
 
+[tool:pytest]
+addopts =
+    --doctest-glob="doc/*.rst"
+    --doctest-modules
+doctest_optionflags =
+    NORMALIZE_WHITESPACE IGNORE_EXCEPTION_DETAIL
```

Comment:
Nope, removing. This might be helpful if someone writes doctests a little _less_ carefully someday.